### DISCUSSION TITLE
WSL Support and Features

### DISCUSSION THREAD

**Discussion by Dharmendra-hub** [zed-industries/zed#6888] [Feature Requests]
Excited to know about integration with Windows Subsystem for Linux

**Comment by cdnspix** [comment]
It is sad that initial support is for MacOS only. MacOS is the only operating system that is not available to poor developers. It's closed-source, very hard to reverse engineer and closely tied to specific hardware. Most people in Latin America had never seen a Mac in their entire lives. And it's impossible to buy affordable Macs. Most developers in poor countries buy very cheap mini PCs in AliExpress for less than 100 USD and only very rich people that own large companies buy Macs, which are considered luxury good.

**Comment by tml** [comment]
Discussion about platform support (Windows / Linux) can be found at https://github.com/zed-industries/zed/issues/5391
